src/job.rs

```diff
@@ -1,10 +1,24 @@
-extern chrono;
+extern crate chrono;
 
 use chrono::prelude::*;
 
-mod job {
-    struct Job<T> {
-        activity: Fn<T>,
-        deadline: DateTime,
+#[derive(Debug)]
+struct Activity {
+    command: String,
```

Comment:
does this need to be an `AsRef` or `OsStr` or what have you for the cases that the command filename isn't _quite_ well formed UTF-8? Especially since on most platforms the command is just a filename.